macros/utils/table_operations/replace_data.sql

```diff
@@ -0,0 +1,43 @@
+{% macro replace_data(relation, sql_query) %}
+    {{ return(adapter.dispatch('replace_data', 'elementary')(relation, sql_query)) }}
+{% endmacro %}
+
+{% macro replace_data_with_table_contents(relation, replacing_relation) %}
+    {% do return(elementary.replace_data(relation, 'select * from {}'.format(replacing_relation))) %}
+{% endmacro %}
+
+{# Databricks / Spark (non-atomic implementation) #}
+{% macro default__replace_data(temporary, relation, sql_query) %}
```

Comment:
No transactions in databricks, and in any case I thought it's good to have a generic default implementation, even if it's not atomic